ocp-templates/mpp/tenant-namespace.yml

```diff
@@ -0,0 +1,37 @@
+apiVersion: template.openshift.io/v1
```

Comment:
Do we you we need to ship this? 
My personal thought, Its part of admin stuff.  Secondly its common for all users of that specific cluster. 
I want to say cluster specific task.

Reply:
removed from the PR